string state_machine_name
string state
